setup.py

```diff
@@ -5,7 +5,8 @@
 
 setup(
     name='anosql',
-    version='1.0.1',
+    setup_requires=['setuptools_scm'],
```

Comment:
Sure, It's indeed not simple as it look like :)

https://github.com/pypa/setuptools_scm/ is maintained by the "Python Packaging Authority" and provide automatic version based on source control (git / hg etc) to provide a single version identifier.

For instance, on top of my head:
```
$ python setup.py --version
1.0.2.dev26+g362701f
```

This mean we are 26 commits on top of the tag 1.0.2 and the commit hash is 362701f (g is for "git").

So to release the 1.1.0 @honza will have to add a tag 1.1.0 and then ``python setup.py --version`` will return exactly "1.1.0".

TL;DR;

```
git tag X.Y.Z
git clean -fxd
python setup.py sdist
python setup.py bdist_wheel
twine upload dist/*
```
When running ``python setup.py sdist``, setuptools_scm will "freeze" the version in the tarbal "PKG-INFO" file.

But this will still require setuptools to pull setuptools_scm when installing. In particular this will fail with very old versions of setuptools (< 12 which is *very* old, but keep in mind that centos 7 still ship 0.9.6...).
To avoid this (and various issues related to old pip/setuptools versions), the state of the art is to also upload "wheel" packages along with the , which is a "pre-installed" package that doesn't require setuptools_scm and will install faster. (Btw when writting this I also added daebcde to build "universal" wheel)

Let me known if setuptools_scm is worth for you, if the release process appear to be too complex I can just drop this particular commit.